File: Fibonacci Sequence - Python 3/fib.py
print ("Fibonacci Sequence Generator"+"\n"+"\n")
x = int(input("Enter the size of sequence"+"\n"))
f = [0,1]
if x == 1:
    f.remove(1)
    print ("The Fibonacci Sequence for the entered number is"+"\n"+str(f))
elif x == 2:
    print ("The Fibonacci Sequence for the entered number is"+"\n"+str(f))
elif x > 2 :
    for i in range(2,x):
        f.append(f[-1]+f[-2])
    print("The Fibonacci Sequence for the entered number is"+"\n"+str(f))
else:
    print("invaild number")

File: Fibonacci Sequence - C Plus Plus/fib.cc
#include <iostream>

using namespace std;

int fibo(int);

int main()
{
    int t;
    cout << "enter number : ";
    cin >> t;
    for (int c = 0; c <= t; c++) {
        cout << fibo(c) << "\t";
    }
    return 0;
}


int fibo(int r)
{
    if (r == 1 || r == 0) {
        return r;
    } else {
        return (fibo(r - 1) + fibo(r - 2));
    }
}
